src/Spinner.class.ts

```diff
@@ -74,12 +76,8 @@ export class Spinner extends EventEmitter {
     const { name = kDefaultSpinnerName, color = null } = options;
 
     this.#spinner = name in cliSpinners ? cliSpinners[name] : cliSpinners[kDefaultSpinnerName];
-    if (color === null) {
-      this.#color = (str: string) => str;
-    }
-    else {
-      this.#color = color in kleur ? kleur[color] : kleur.white;
-    }
+
+    this.#color = color !== null && kAvailableColors.has(color) ? color : "white";
```

Comment:
```suggestion
    this.#color = kAvailableColors.has(color) ? color : "white";
```
Should we allow array of colors since we allow modifiers ?
For instance `{ color: ["red", "bold"] }`
@fraxken WDYT ?